composer.json

```diff
@@ -18,7 +19,7 @@
     },
 
     "scripts": {
-        "phpstan": "tools/phpstan analyze",
+        "phpstan": "vendor/bin/phpstan analyze",
```

Comment:
I changed this back to `tools/phpstan analyze` and it seemingly loads the PHPUnit extension just fine, at least I did get proper errors when using the framework ruleset and trying `assertSame(true, ...)`.

Reply:
I changed this for a reason. Composer enforces the installation of compatible versions between PHPStan and its extensions. If you roll back this change, you'll have PHPStan installed twice for no reason.